builds.sh

```diff
@@ -19,13 +19,27 @@ shell="$(jq -r '
 ' "$json")"
 eval "$shell"
 
+_resolveRemoteArch() {
+	local img="$1"; shift
+
+	local arches
+	if ! arches="$(bashbrew remote arches --json "$img" 2>/dev/null)"; then # TODO somehow differentiate errors like 404 / 403, "insufficient_scope" from other bashbrew errors here so we can stop eating stderr
+		printf 'null'
+		return
+	fi
+
+	echo "$arches"
+}
+
 # resolve an image reference to an architecture-specific imageId (digest of the image manifest)
 _resolve() {
 	local img="$1"; shift
 	local arch="$1"; shift
+	local bashbrewRemoteArches=
```

Comment:
Now that we're always passing a value (either a JSON object or the JSON null value), this can be reverted and the conditional can be simply `if [ "$bashbrewRemoteArches" = 'null' ]; then printf 'null'; return; fi`